@@@ A - classifier: SVC(C=1.0, break_ties=False, cache_size=200, class_weight='balanced', coef0=0.0,
    decision_function_shape='ovr', degree=3, gamma='scale', kernel='linear',
    max_iter=-1, probability=True, random_state=None, shrinking=True, tol=0.001,
    verbose=False)
@@@ B - threshold stop at: 1
@@@ C - Repeat number: 1
/Users/author1/anaconda3/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:463: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint8 = np.dtype([("qint8", np.int8, 1)])
/Users/author1/anaconda3/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:464: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint8 = np.dtype([("quint8", np.uint8, 1)])
/Users/author1/anaconda3/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:465: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint16 = np.dtype([("qint16", np.int16, 1)])
/Users/author1/anaconda3/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:466: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint16 = np.dtype([("quint16", np.uint16, 1)])
/Users/author1/anaconda3/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:467: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint32 = np.dtype([("qint32", np.int32, 1)])
@@@ D - common_header:  {'F71-rjung@apache.org', 'F110', 'F3-NA', 'F15-public', 'F71-yoavs@apache.org', 'F3-Object', 'F68', 'F3-Member', 'F15-private', 'F41', 'F71-mturk@apache.org', 'category', 'F71-slaurent@apache.org', 'F15-NA', 'F71-billbarker@apache.org', 'F126', 'F117', 'F71-jfclere@apache.org', 'F65', 'F3-InputStream', 'F3-String[]', 'F71-markt@apache.org', 'F22', 'F71-schultz@apache.org', 'F123', 'F115', 'F71-fhanik@apache.org', 'F71-costin@apache.org', 'F104', 'F71-kkolinko@apache.org', 'F3-String', 'F77', 'F120', 'F3-Class<?>[]', 'F71-timw@apache.org', 'F3-Class[]', 'F3-ObjectName', 'F71-remm@apache.org', 'F3-byte[]', 'F3-boolean', 'F15-protected', 'F116', 'F3-int', 'F71-fcarrion@apache.org', 'F3-ELContext', 'F25', 'F3-null', 'F101', 'F71-idarwin@apache.org', 'F71-funkman@apache.org', 'F3-Object[]', 'F72', 'F3-Session', 'F3-FileInputStream', 'F3-Integer', 'F3-long', 'F71-jhook@apache.org', 'F71-pero@apache.org', 'F3-Long', 'F105', 'F71-kfujino@apache.org'}
Traceback (most recent call last):
  File "baseline.py", line 217, in <module>
    seed=42 + i, clf=clf, interesting_path=argv )
  File "baseline.py", line 43, in main
    training_x, training_y, testset_x, testset_y, training_canonical_ids, testing_canonical_ids = cleaning.data_clean(path, [interesting_path], seed)
  File "/Users/author1/repos/SA_public/src/cleaning.py", line 340, in data_clean
    testset_y = testing_trim.iloc[:, -1]
  File "/Users/author1/anaconda3/lib/python3.6/site-packages/pandas/core/indexing.py", line 1472, in __getitem__
    return self._getitem_tuple(key)
  File "/Users/author1/anaconda3/lib/python3.6/site-packages/pandas/core/indexing.py", line 2013, in _getitem_tuple
    self._has_valid_tuple(tup)
  File "/Users/author1/anaconda3/lib/python3.6/site-packages/pandas/core/indexing.py", line 222, in _has_valid_tuple
    self._validate_key(k, i)
  File "/Users/author1/anaconda3/lib/python3.6/site-packages/pandas/core/indexing.py", line 1957, in _validate_key
    self._validate_integer(key, axis)
  File "/Users/author1/anaconda3/lib/python3.6/site-packages/pandas/core/indexing.py", line 2009, in _validate_integer
    raise IndexError("single positional indexer is out-of-bounds")
IndexError: single positional indexer is out-of-bounds
